backend/YouTubeV2.Application/DTO/PlaylistDTOS/PlaylistDto.cs

```diff
@@ -5,6 +5,7 @@
 using System.Threading.Tasks;
```

Comment:
remove all unused usings. (ctrl + on one useing and choose remove from solution or something like that)